util/bitstream_test.go

```diff
@@ -16,6 +17,29 @@ type testDefinition struct {
 	value  uint64 // The value we expect the function to return (64 bit to allow for future functions that extract larger ints)
 }
 
+var test2Bits = []testDefinition{
```

Comment:
My error PR revamps how we are doing this testing. You may want to look at that and alter this to match.